Library-SMTLIB/src/de/uni_freiburg/informatik/ultimate/logic/Theory.java

```diff
@@ -856,6 +856,32 @@ public Sort getResultSort(final String[] indices, final Sort[] paramSorts, final
 				return paramSorts[0];
 			}
 		}
+		class Bv2IntFunction extends FunctionSymbolFactory {
+			public Bv2IntFunction(final String name) {
+				super(name);
+				assert name.equals("bv2int") : "Wrong name: " + name;
+			}
+
+			@Override
+			public Sort getResultSort(final String[] indices, final Sort[] paramSorts, final Sort resultSort) {
+				return mNumericSort;
+			}
+		}
+		class Int2BvFunction extends FunctionSymbolFactory {
+			public Int2BvFunction(final String name) {
+				super(name);
+				assert name.equals("int2bv") : "Wrong name: " + name;
+			}
+
+			@Override
+			public Sort getResultSort(final String[] indices, final Sort[] paramSorts, final Sort resultSort) {
+				assert indices.length == 1;
```

Comment:
These should be a real if as above returning null if the conditions are wrong.

Reply:
I do not complete understand you message but I addressed this in https://github.com/ultimate-pa/smtinterpol/commit/c38d53bd77c52e25e968a50f6f1ad091b3c796a5